doc/source/getting-started/administration.rst

```diff
@@ -0,0 +1,161 @@
+Project approval and public release
+===================================
+Most of the projects in PyAnsys expose the functionality of ANSYS, Inc.
+products. Due to intellectual property reasons, the public release of a PyAnsys
+library must undergo through a project approval process.
+
+
+Approval process
+----------------
+The approval process is divided into three different parts:
+
+.. grid:: 3
+    
+    .. grid-item-card:: :octicon:`file-badge` Managerial
+       :link: managerial
+       :link-type: ref
+
+        Verifies that direct managers, GM/VP, CT or CTO are aware about the project and approve it.
+
+    .. grid-item-card:: :octicon:`law` Legal
+       :link: legal
+       :link-type: ref
+
+        Verifies that the project ships with the right legal framework to protect ANSYS, Inc. intellectual property.
+
+    .. grid-item-card:: :octicon:`gear` Technical
+       :link: technical
+       :link-type: ref
+
+        Verifies that the project is compliant with ANSYS, Inc. software guidelines and best practices.
+
+
+.. important::
+
+    An approval from each part is required to release a project into the public.
+
+
+When releasing a project to the public, expect to:
+
+* Coordinate with the product line development team, if applicable.
+* Maintain the project by means of fixing bugs and providing support for new releases.
+* Ensure the good reputation of ANSYS, Inc. in the open source community.
+
+
+Managerial
+^^^^^^^^^^
+The managerial part of the approval process guarantees that direct manager (DM),
+general managers (GM), vice presidents (VP), and chief technolofy officer (CTO)
+are aware about the existence and the status of the project.
+
+The amount of administrative reviews and approvals depends on the nature of the
+project. A project can be classified in to the following categories:
+
+.. grid:: 3
+    
+    .. grid-item-card:: :octicon:`repo` Documentation projects
+
+        Documentation projects require direct manager approval, legal
```

Comment:
Do we use DM instead of direct manager since you defined it earlier? You are using the other abbreviations that you defined earlier. If you don't want to use DM in the text, maybe don't define it earlier?

Reply:
I finally decided not to use any abbreviations for these names. This makes them more explicit.